client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -78,11 +78,14 @@ function ReaderSubscriptionListItem( {
 					</span>
 				}
 			{ siteUrl && (
-				<div className="reader-subscription-list-item__site-url">
-					<a href={ siteUrl } target="_blank" rel="noopener noreferrer">
-						{ stripUrl( siteUrl ) }
-					</a>
-				</div>
+				<a
+					href={ siteUrl }
+					target="_blank"
+					rel="noopener noreferrer"
+					className="reader-subscription-list-item__site-url"
+				>
+					{ stripUrl( siteUrl ) }
```

Comment:
I wonder if we should rename this method now. I read stripUrl as "it strips all URLs from a string". How about `getUrlForDisplay` or similar? `stripUrlProtocol`?